README.md

```diff
@@ -205,19 +207,23 @@ the regional metaweb [@Saravia2022Ecological]. Without clear documentation, it
 can be challenging to know if published probabilistic interaction webs describe
 local or regional interactions (@Tbl:prob provides examples of studies employing
 both types of probabilistic interaction networks), or if so-called probabilities
-are in reality *interaction scores* (i.e., non-probabilistic quantitative
-interactions analogous to interaction strengths). When probabilistic regional
+are in reality a form of interaction score (i.e., non-probabilistic quantitative
```

Comment:
L210: Interactions scores are not mentioned again so I think it's better if we rephrase a bit and don't highlight them in italic. Otherwise it feels like introducing a third type of representation (with metawebs and local webs, also in italic here) but then don't cover it further.